### PR TITLE
added a square waveform generator

### PR DESCRIPTION
**Pull request by brentru**
Generates a single square wave of sample_length size

### CODE REVIEW

File: adafruit_waveform/square.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* Author(s): Scott Shawcroft, BrentRu
+"""
+
+import array
+
+
+def square_wave(sample_length = 2):
+    """Generate a single square wave of sample_length size"""
+    s = array.array("H", [0] * sample_length)
+    for i in range(sample_length / 2):
```
Comment (tannewt) [inline adafruit_waveform/square.py]: I think you'll want `sample_length // 2` to make sure you do truncated division.

**Review by tannewt — CHANGES_REQUESTED** [review]
Please fix lint as well.

File: adafruit_waveform/square.py (hunk 1 above)
Comment (brentru) [reply adafruit_waveform/square.py]: Added!

**Review by tannewt — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**